sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -650,6 +718,15 @@ private void finishConfiguration() {
         mLogSerializer.addLogFactory(StartServiceLog.TYPE, new StartServiceLogFactory());
         mLogSerializer.addLogFactory(CustomPropertiesLog.TYPE, new CustomPropertiesLogFactory());
         mChannel = new DefaultChannel(mApplication, mAppSecret, mLogSerializer, mHandler);
+
+        /* Complete set maximum storage size future if starting from app. */
+        if (configureFromApp) {
+            applyStorageMaxSize();
+        } else {
+
+            /* If from library, we apply storage size only later, we have to try using the default value in the mean time. */
+            mChannel.setMaxStorageSize(DEFAULT_MAX_STORAGE_SIZE_IN_BYTES);
```

Comment:
Can you explain me about this line? Why do we need to set a storage limit to default value when it is started from lib?

Reply:
Because we still store logs when started from a library, but we don't allow it to be set manually when starting from a library, so we use the default value.